app/web/src/components/ModelingDiagram/ModelingDiagram.vue

```diff
@@ -2538,15 +2538,9 @@ const drawEdgePossibleTargetSocketKeys = computed(() => {
     if (fromSocket.def.direction === possibleToSocket.def.direction)
       return false;
 
-    const isManagedSchema =
```

Comment:
This logic decides which management sockets we should show as "open" while you are dragging an edge. The logic says "you may not drag from a management socket to another management socket unless it's in managed schemas." It then goes on to disallow them if the connection annotations do not match, which is probably how we implicitly disallow management <-> non-management connections since most (all?) sockets have annotations.

In the absence of management schemas, this logic just says "management sockets only connect to other management sockets."